### PR TITLE
Example CodeReviewHub Review PR

### PR DESCRIPTION
**Pull request by tom-dalton-fanduel**
This is just an example PR so we can see how CodeReviewHub works.
<a href='#crh-start'></a><a href='#crh-data-%7B%22approved%22%3A%20%7B%7D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/tom-dalton-fanduel/twemproxy/pull/1%23issuecomment-175055349%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%3Ashipit%3A%22%2C%20%22created_at%22%3A%20%222016-01-26T14%3A48%3A50Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/10129198%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/AlastairAtFanduel%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3Ashipit%3A%3Agun%3A%22%2C%20%22created_at%22%3A%20%222016-01-26T14%3A48%3A56Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/10129198%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/AlastairAtFanduel%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%3Ca%20href%3D%27%23crh-update-none%27%3E%3C/a%3E%22%2C%20%22created_at%22%3A%20%222016-01-26T14%3A58%3A55Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7839062%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/tom-dalton-fanduel%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%20c55daaf0bfa6333753ff74c27699fa1c2f53b381%20configure.ac%205%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/tom-dalton-fanduel/twemproxy/pull/1%23discussion_r50842824%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Just%20some%20comment...%22%2C%20%22created_at%22%3A%20%222016-01-26T14%3A46%3A01Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7839062%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/tom-dalton-fanduel%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%3Ca%20href%3D%27%23crh-update-none%27%3E%3C/a%3E%22%2C%20%22created_at%22%3A%20%222016-01-26T14%3A58%3A55Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7839062%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/tom-dalton-fanduel%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20configure.ac%3AL1-8%22%7D%2C%20%22Pull%20c55daaf0bfa6333753ff74c27699fa1c2f53b381%20src/proto/nc_redis.c%205%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/tom-dalton-fanduel/twemproxy/pull/1%23discussion_r50843235%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22Another%20comment%22%2C%20%22created_at%22%3A%20%222016-01-26T14%3A48%3A59Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/4276370%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/gando999%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%3Ca%20href%3D%27%23crh-update-complete%27%3E%3C/a%3E%22%2C%20%22created_at%22%3A%20%222016-01-26T14%3A58%3A56Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/7839062%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/tom-dalton-fanduel%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20src/proto/nc_redis.c%3AL1920-1929%22%7D%2C%20%22Pull%20c55daaf0bfa6333753ff74c27699fa1c2f53b381%20tests/test_redis/test_lua_error_return.py%204%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/tom-dalton-fanduel/twemproxy/pull/1%23discussion_r50843314%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22So%20many%20bugs%22%2C%20%22created_at%22%3A%20%222016-01-26T14%3A49%3A33Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/10129198%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/AlastairAtFanduel%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20tests/test_redis/test_lua_error_return.py%3AL1-34%22%7D%7D%2C%20%22processed%22%3A%20%5B%22https%3A//github.com/tom-dalton-fanduel/twemproxy/pull/1%23issuecomment-175055349%22%2C%20%22https%3A//github.com/tom-dalton-fanduel/twemproxy/pull/1%23issuecomment-175055417%22%2C%20%22https%3A//github.com/tom-dalton-fanduel/twemproxy/pull/1%23issuecomment-175059058%22%2C%20%22https%3A//github.com/tom-dalton-fanduel/twemproxy/pull/1%23discussion_r50842824%22%2C%20%22https%3A//github.com/tom-dalton-fanduel/twemproxy/pull/1%23discussion_r50843235%22%2C%20%22https%3A//github.com/tom-dalton-fanduel/twemproxy/pull/1%23discussion_r50843314%22%2C%20%22https%3A//github.com/tom-dalton-fanduel/twemproxy/pull/1%23discussion_r50844766%22%2C%20%22https%3A//github.com/tom-dalton-fanduel/twemproxy/pull/1%23discussion_r50844767%22%5D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [ ] <a href='#crh-comment-Pull c55daaf0bfa6333753ff74c27699fa1c2f53b381 tests/test_redis/test_lua_error_return.py 4'></a> &nbsp;<b><a href='https://github.com/tom-dalton-fanduel/twemproxy/pull/1#discussion_r50843314'>File: tests/test_redis/test_lua_error_return.py:L1-34</a></b>
- <a href='https://github.com/AlastairAtFanduel'></a> So many bugs
- [x] <a href='#crh-comment-Pull c55daaf0bfa6333753ff74c27699fa1c2f53b381 configure.ac 5'></a> &nbsp;<b><a href='https://github.com/tom-dalton-fanduel/twemproxy/pull/1#discussion_r50842824'>File: configure.ac:L1-8</a></b>
- <a href='https://github.com/tom-dalton-fanduel'></a> Just some comment...
- [x] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/tom-dalton-fanduel/twemproxy/pull/1#issuecomment-175055349'>General Comment</a></b>
- <a href='https://github.com/AlastairAtFanduel'></a> :shipit:
- <a href='https://github.com/AlastairAtFanduel'></a> :shipit::gun:
- [x] <a href='#crh-comment-Pull c55daaf0bfa6333753ff74c27699fa1c2f53b381 src/proto/nc_redis.c 5'></a> &nbsp;<b><a href='https://github.com/tom-dalton-fanduel/twemproxy/pull/1#discussion_r50843235'>File: src/proto/nc_redis.c:L1920-1929</a></b>
- <a href='https://github.com/gando999'></a> Another comment

<a href='https://www.codereviewhub.com/tom-dalton-fanduel/twemproxy/pull/1?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/tom-dalton-fanduel/twemproxy/pull/1?approve=1'></a>&nbsp;<a href='https://github.com/tom-dalton-fanduel/twemproxy/pull/1'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by tom-dalton-fanduel** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/tom-dalton-fanduel/twemproxy/pull/1'>Refresh</a>.<a href='#crh-update-none'></a>

File: configure.ac [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Define the package version numbers and the bug reporting address
 m4_define([NC_MAJOR], 0)
 m4_define([NC_MINOR], 4)
-m4_define([NC_PATCH], 1)
+m4_define([NC_PATCH], 1-fd1)
```
Comment (tom-dalton-fanduel) [inline configure.ac:4]: Just some comment...

**Comment by AlastairAtFanduel** [comment]
:shipit:

**Comment by AlastairAtFanduel** [comment]
:shipit::gun:

File: src/proto/nc_redis.c [hunk 2]
```diff
@@ -1920,6 +1920,9 @@ redis_parse_rsp(struct msg *r)
 
                     break;
                 }
+                if (ch == CR) {
+                    p -= 1;
```
Comment (gando999) [inline src/proto/nc_redis.c:1924]: Another comment

File: tests/test_redis/test_lua_error_return.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+#coding: utf-8
+
+from unittest import TestCase
```
Comment (AlastairAtFanduel) [inline tests/test_redis/test_lua_error_return.py:4]: So many bugs

File: configure.ac (hunk 1 above)
Comment (tom-dalton-fanduel) [inline configure.ac:4]: :+1:<a href='#crh-update-none'></a>

**Comment by tom-dalton-fanduel** [comment]
:+1:<a href='#crh-update-none'></a>

File: src/proto/nc_redis.c (hunk 2 above)
Comment (tom-dalton-fanduel) [inline src/proto/nc_redis.c:1924]: :+1:<a href='#crh-update-complete'></a>

**Pull request closed without merging**